### DISCUSSION TITLE
Public map resolution

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#440] [General]
Now public map generates page with resolution 1920x900 pixels. This makes some difficulty to embed map into web pages (especially for mobile device view).
Is it possible to customize public map resolution?

**Comment by HB9HIL** [comment]
Hello @R1BLH 

You can adjust the resolution with some iframe/css magic:

```
<iframe src="https://[URL]/index.php/visitor/exportmap/[SLUG]" width="800" height="600" style="transform: scale(0.5); transform-origin: 0 0;" frameborder="0"></iframe>
```

Play around with `width`, `height` and `scale`

I hope this helps

**Reply by R1BLH** [reply]
> Play around with `width`, `height` and `scale`

Already did this. 
in the нeader
```
<stiyle> 
....
.map:not(:first-child) {
margin-block-start: 1em;
}
.map {
text-align: center;
    -moz-transform: scale(0.5, 0.5); 
    -webkit-transform: scale(0.5, 0.5); 
    -o-transform: scale(0.5, 0.5);
    -ms-transform: scale(0.5, 0.5);
    transform: scale(0.5, 0.5); 
  -moz-transform-origin: top center;
  -webkit-transform-origin: top center;
  -o-transform-origin: top center;
  -ms-transform-origin: top center;
  transform-origin: top center;   
}
.map > iframe {
width: 1200px;
max-width: 100%;
height: max(300px, 100vh - 20px - 2rem);
}
...
</style>

```
in the body:
`<div class="map"><iframe src="[https://wavelog.cats-home.net/index.php/visitor/exportmap/R1BLH](view-source:https://wavelog.cats-home.net/index.php/visitor/exportmap/R1BLH)"></iframe></div>`

result is here: https://r1blh.cats-home.net/
But the scale operation takes a lot of resources on the weak PCs.

**Reply by HB9HIL** [reply]
the leaflet map is completely rendered in the browser, so no matter what you do, it will always a bit slow on weak pc's. There is not really something we can do about. Even if there would be an option to change the resolution of the map, I doubt it would gain much performance due to the javascript of Leaflet. 

In the current `dev` branch are some changes, which makes the whole loading of the map a bit faster (#438). This will help. I'm sorry. Currently I don't know about plans to implement a resolution logic. If you have a solution you can make a PR and we will see. 
:)

**Reply by R1BLH** [reply]
Understandable. Thank you!
